examples/flutter_gallery/web/index.html

```diff
@@ -1,12 +1,25 @@
-<!DOCTYPE HTML>
+<!DOCTYPE html>
 <!-- Copyright 2014 The Flutter Authors. All rights reserved.
 Use of this source code is governed by a BSD-style license that can be
 found in the LICENSE file. -->
 <html>
-    <head>
-        <title>Flutter Gallery</title>
-    </head>
-    <body>
-        <script src="main.dart.js"></script>
-    </body>
+<head>
+  <meta charset="UTF-8">
+  <meta content="IE=Edge" http-equiv="X-UA-Compatible">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
We already add this meta tag: https://github.com/flutter/engine/blob/master/lib/web_ui/lib/src/engine/dom_renderer.dart#L379

Reply:
I didn't know that worked, but neat!

Reply:
Removed